datocms.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "profiles": {
+    "default": {
+      "logLevel": "NONE",
+      "migrations": {
+        "directory": "./migrations",
+        "modelApiKey": "schema_migration",
+        "tsconfig": "./tsconfig.json"
+      }
+    }
+  }
+}
```

Comment:
When trying to create a migration from the changes done on the sandbox it said I was missing this file, I created it following [this instructions](https://www.datocms.com/docs/scripting-migrations/installing-the-cli#setting-up-a-cli-profile). 
But this should be in Selwyn's PR if we want to use it.

Reply:
Yeah I think this will be replaced by the one in Selwyn's branch once this branch is up to date again.